src/extensions/custom-view/bootstrap-table-custom-view.js

```diff
@@ -62,8 +77,8 @@ $.BootstrapTable = class extends $.BootstrapTable {
     if (this.options.customView && this.options.showCustomView) {
       this.buttons = Object.assign(this.buttons, {
         customView: {
-          text: this.options.formatToggleCustomView(),
-          icon: this.options.icons.customView,
+          text: this.options.customViewDefaultView ? this.options.formatToggleCustomViewOff() : this.options.formatToggleCustomViewOn(),
+          icon: this.options.customViewDefaultView ? this.options.icons.customViewOn : this.options.icons.customViewOff,
           event: this.toggleCustomView,
           attributes: {
             'aria-label': this.options.formatToggleCustomView(),
```

Comment:
As already commented, i guess `formatToggleCustomView` can be removed and replaced with the correct text (based on the current value).

Reply:
ok, removed.